scripts/binary/util/packages.ts

```diff
@@ -0,0 +1,176 @@
+import _ from 'lodash'
+import fs from 'fs-extra'
+import path from 'path'
+// we wrap glob to handle EMFILE error
+import Promise from 'bluebird'
```

Comment:
Do we need bluebird here?

Reply:
It looks like we're just using it for Promise.all which the node promise implementation should handle.

Reply:
Again, was more of a find/replace from `packages.js`, but no, it doesn't look like we do